media/js/teach/smarton.js

```diff
@@ -0,0 +1,258 @@
+            if (direction === 'down') {
+                $pageHeader.find('.nav-steps li').removeClass();
+                $('#nav-step-' + current).addClass('current');
+            }
+            else {
+                $pageHeader.find('.nav-steps li').removeClass();
+                $('#nav-step-' + previous).addClass('current');
+            }
+        };
+    }
+
+    if ($pageHeader.length > 0) {
+        //Fire the waypoints for each section, passing classes for the current and previous sections
+        $('#ask').waypoint(navState('ask', 'ask'), { offset: $pageHeader.height() + 10 });
+        $('#know').waypoint(navState('know', 'ask'), { offset: $pageHeader.height() + 10 });
+        $('#do').waypoint(navState('do', 'know'), { offset: $pageHeader.height() + 10 });
+        $('#chat').waypoint(navState('chat', 'do'), { offset: $pageHeader.height() + 10 });
+    }
+
+    // Draw animated SVG pie charts
+    // Requires /libs/circles.min.js - http://lugolabs.com/circles
+    function drawCircle(circleId, circleVal) {
+        Circles.create({
+          id:                  circleId,
+          value:               circleVal,
+          radius:              70,
+          maxValue:            100,
+          width:               12,
+          text:                function(value) {return value + '%';},
+          duration:            850,
+          wrpClass:            'circles-wrap',
+          textClass:           'circles-text',
+          valueStrokeClass:    'circles-valueStroke',
+          maxValueStrokeClass: 'circles-maxValueStroke',
+          styleWrapper:        false,
+          styleText:           false,
+        });
+    }
+
+    // Trigger the chart when it scrolls into view
+    $('.sidebar-chart').each(function() {
+        var stage = $(this).children('.chart-stage');
+        var circleId = stage.attr('id');
+        var circleVal = stage.data('chart-value');
+        var chart = new Waypoint({
+            element: stage,
+            handler: function(direction) {
+                if (direction === 'down') {
+                    drawCircle(circleId, circleVal);
+                }
+            },
+            offset: '100%'
+        });
+    });
+
+    // Play videos in a modal
+    $('a.video-open').attr('role', 'button').on('click', function(e) {
+        e.preventDefault();
+
+        var $this = $(this);
+        var embedSrc = $this.data('embed-src');
+        var embedWrap = $('#' + $this.data('element-id'));
+
+        Mozilla.Modal.createModal(this, embedWrap, {
+            title: '',
+            onCreate: function() {
+                embedWrap.find('.modal-video-frame').attr('src', embedSrc);
+            },
+            onDestroy: function() {
+                embedWrap.find('.modal-video-frame').removeAttr('src');
+            }
+        });
+
+        // Track the click and record the video ID
+        window.dataLayer.push({
+            event: 'smarton-interactions',
```

Comment:
Nit: GA event calls should have quotes, e.g.

    window.dataLayer.push({
        'event': 'smarton-interactions',
        'interaction': 'video modal open',
        'video': $this.data('element-id')
    });

Same for here and elsewhere in this file.